packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -93,6 +85,14 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
         this.virtualizer?.scrollToIndex(index, options);
     }
 
+    public updateRowHeight(): void {
+        this.updatePageSize();
+        if (this.virtualizer) {
+            this.updateVirtualizer();
+            this.virtualizer.measure();
```

Comment:
Note: After just the `updateVirtualizer()` call, my manual testing showed issues with PageUp/ PageDown behavior after changing the `controlHeight`/`borderWidth` calls (like PageDown scrolling down, but then the focused row was out of the visible scroll area).  
This additional `virtualizer.measure()` call gets the TanStack virtualizer to do the necessary updates and resolves that issue.